py/picca/tests/delta_extraction/expected_flux_tests.py

```diff
@@ -170,14 +170,14 @@ def test_dr16_expected_flux_compute_continuum_lin(self):
             elif continua.get(forest.los_id) is None:
                 self.assertTrue(forest.continuum is None)
             else:
-                if not np.allclose(forest.continuum, continua.get(forest.los_id)):
+                if not np.allclose(forest.continuum[forest.ivar>0], continua.get(forest.los_id)):
```

Comment:
I'd rather update the test suite with the correct changes than to keep this potentially faulty check

Reply:
These test code changes were to make sure masking behaved the same as before. Since new files have more pixels, the array sizes are different. But these codes would be removed if baseline files are replaced with new ones. I think the changes are safe at this points, so I'll replace test files and use them going forward with the comments